src/app/entities/user/user.entity.ts

```diff
@@ -1,24 +1,25 @@
-import { Account } from 'src/app/account/entities/account.entity';
-import { Entity, Column, PrimaryGeneratedColumn, OneToMany } from 'typeorm';
+import {Account} from '../account/account.entity';
+import {Entity, Column, PrimaryGeneratedColumn, OneToMany} from 'typeorm';
 
+//TODO: Use class validator
```

Comment:
I thought class validator should be used for incoming data like InputType or ArgsType, not entities

since name, email and password are already validated at this point, and createdDate, isActive and id are added by the DB

Reply:
I think we should add the validation on the Database layer rather than the GraphQL one